dc-model/src/main/java/de/digitalcollections/model/identifiable/alias/LocalizedUrlAliasesBuilder.java

```diff
@@ -0,0 +1,15 @@
+package de.digitalcollections.model.identifiable.alias;
+
+public class LocalizedUrlAliasesBuilder {
```

Comment:
`LocalizedUrlAliases` hat doch schon einen Konstruktor, dem man mehrere `UrlAlias`se übergeben kann (varargs). Die `add`-Methode ist ebenso varargs. Könnte man da nicht auf diesen Builder verzichten?

Reply:
Wäre OK, dann müßte ich lediglich einen Test in Cudami umbauen

Reply:
done

Reply:
:+1: